pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -43,6 +43,8 @@ const (
 	crioHooksConfigDir  = "/etc/containers/oci/hooks.d"
 	crioHooksConfig     = "99-high-performance-hooks"
 	crioTemplateRpsMask = "RpsMask"
+	udevRulesDir        = "/etc/udev/rules.d"
+	udevRpsRule         = "99-netdev-rps"
```

Comment:
this seems to be used just once in line 165 below. If so why can't we just call this "99-netdev-rps.rules"  ?

Reply:
I have this weird rule to use the same coding standard in the file, especially the micro patterns, so readers would not wonder why is different. I agree there is no other need for that.
I am OK one way or the other.

Reply:
I'm ok following the existing conventions, so we can keep it this way.